strapi/strapi
# feat: add vite

### What does it do?

* adds a first implementation of using `vite` as a bundler

### Why is it needed?

* We want to move to use Vite full time in V5

### How to test it?

* run yarn setup (to make sure you have built packages etc.)

*build*
* go to an example
* run `yarn build --bundler=vite`
* run `yarn start`
* Should see the admin panel work!

*develop*
* go to an example
* run `yarn develop --watch-admin --bundler=vite`
* Should see the admin panel work!

### Related issue(s)/PR(s)

* resolves CONTENT-1625
* resolves CONTENT-1626
* resolves CONTENT-1627
* resolves CONTENT-1650
* resolves CONTENT-2082
* resolves https://github.com/strapi/rfcs/issues/30
